### ISSUE TITLE
Perceptual loss issue

### ISSUE THREAD

**Issue by dhruva15026** [SeitaroShinagawa/chainer-partial_convolution_image_inpainting#10]
Hi,
It seems there is an issue with the perceptual loss.
In https://github.com/SeitaroShinagawa/chainer-partial_convolution_image_inpainting/blob/master/updater.py#L14, it is :  

    layers = list(hout_dict.keys())
    layer_name =  layers[0]
    loss = F.mean_absolute_error(hout_dict[layer_name],hgt_dict[layer_name])
    loss += F.mean_absolute_error(hout_dict[layer_name],hgt_dict[layer_name])
    for layer_name in layers[1:]: 
        loss += F.mean_absolute_error(hcomp_dict[layer_name],hgt_dict[layer_name])
        loss += F.mean_absolute_error(hcomp_dict[layer_name],hgt_dict[layer_name])

while in actual it should be:

    layers = list(hout_dict.keys())
    layer_name =  layers[0]
    loss = F.mean_absolute_error(hout_dict[layer_name],hgt_dict[layer_name])
    loss += F.mean_absolute_error(hcomp_dict[layer_name],hgt_dict[layer_name])
    for layer_name in layers[1:]: 
        loss += F.mean_absolute_error(hout_dict[layer_name],hgt_dict[layer_name])
        loss += F.mean_absolute_error(hcomp_dict[layer_name],hgt_dict[layer_name])

**Comment by SeitaroShinagawa** [comment]
Hi @dhruva15026,  
Thanks for your report! I have fixed it in https://github.com/SeitaroShinagawa/chainer-partial_convolution_image_inpainting/commit/851a977ab697877284490e9fe6f4e08cc169c978.
